translations/base-it.yaml

```diff
@@ -502,181 +504,182 @@ buildings:
 
     painter:
         default:
-            name: &painter Verniciatore
-            description: &painter_desc Colora l'intera forma dell'ingresso sinistro con il colore dell'ingresso destro.
+            name: &painter Verniciatrice
+            description: &painter_desc Colora l'intera forma dall'ingresso sinistro con il colore dall'ingresso destro.
         double:
-            name: Verniciatore (Doppio)
-            description: Colors the shapes on the left inputs with the color from the top input.
+            name: Verniciatrice (2x)
+            description: Colora le forme dagli ingressi sinistri con il colore dall'ingresso destro.
         quad:
-            name: Verniciatore (Quadruplo)
-            description: Allows to color each quadrant of the shape with a different color.
+            name: Verniciatrice (4x)
+            description: Consente di colorare ogni quadrante della forma con un colore diverso.
         mirrored:
             name: *painter
             description: *painter_desc
 
     trash:
         default:
             name: &trash Cestino
-            description: Accetta ingressi da tutti i lati e li distrugge. per sempre.
+            description: Accetta oggetti da tutti i lati e li distrugge. Per sempre.
 
         storage:
-            name: Storage
-            description: Stores excess items, up to a given capacity. Can be used as an overflow gate.
+            name: Stoccaggio
+            description: Immagazzina gli oggetti in eccesso, fino ad una certa capacità. Può essere usato come varco per le eccedenze #Literal translation, quite ugly, might need rephrasing
```

Comment:
I'm not sure whether your comment will be visible ingame or not. Might wait for approval of @ tobspr

Reply:
Don't know either, I am not familiar at all with Yaml

Reply:
It will I *think*, I'll remove it